src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import lotto.view.OutputView;
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers; //로또 갯수 저장필드
+    private static final int LOOTO_AMOUNT = 1000;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    public static Lotto createRandomLotto() {
+        List<Integer> lottoNumbers = Randoms.pickUniqueNumbersInRange(1, 45, 6);
+        Collections.sort(lottoNumbers);
+        return new Lotto(lottoNumbers);
+    }
+    public List<Integer> getNumbers() {
+        return numbers;
+    }
+
+    public static void validatePurchaseAmount(int amount) {
+        if (amount < LOOTO_AMOUNT) {
+            OutputView.printMinimumPurchaseMessage();
+            throw new IllegalArgumentException();
```

Comment:
예외발생시 출력을 따로 분리하지 않아도 됩니다.
IllegalArgumentException 클래스를 코드 레벨에서 들여다 보고 어떤 특징이 있는지 알고 사용하면 좋습니다.

```suggestion
            throw new IllegalArgumentException("에러 메시지 작성 공간");
```